common/views/components/BasePage/EventPage.js

```diff
@@ -5,13 +5,77 @@ import BaseHeader from '../BaseHeader/BaseHeader';
 import Body from '../Body/Body';
 import Contributors from '../Contributors/Contributors';
 import WobblyBackground from '../BaseHeader/WobblyBackground';
+import Tags from '../Tags/Tags';
+import Icon from '../Icon/Icon';
 import {UiImage} from '../Images/Images';
 import type {Event} from '../../../model/events';
+import {spacing, font} from '../../../utils/classnames';
+import camelize from '../../../utils/camelize';
+import {formatAndDedupeOnDate, formatAndDedupeOnTime, joinDateStrings} from '../../../utils/format-date';
 
 type Props = {|
   event: Event
 |}
 
+function dateInfo(times) {
+  return (
+    <Fragment>
```

Comment:
Do we want to be using these?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/views/components/DateRange/DateRange.js

Or creating new ones to make sure we're doing dates across the site consistently?

Reply:
Good question, will discuss with @GarethOrmerod